### PR TITLE
Update templates from eventing

### PR DESCRIPTION
**Pull request by grantr**
Copying https://github.com/knative/eventing/pull/2601 plus the rest of the eventing templates, since these haven't been updated when the eventing ones changed.

/cc @vaikas @lionelvillard

### CODE REVIEW

File: ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: ''
+labels: kind/feature-request
+assignees: ''
+
+---
+
+**Problem**
+A short explanation of the problem, including relevant restrictions.
+
+**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)**
```
Comment (mattmoor) [inline ISSUE_TEMPLATE/feature-request.md]: Remove trailing whitespace:
```suggestion
**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)**
```

File: ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+**Problem**
+A short explanation of the problem, including relevant restrictions.
+
+**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
+Which persona is this feature for?
+
+**Exit Criteria**
+A measurable (binary) test that would indicate that the problem has been resolved.
+
+**Time Estimate (optional):**
```
Comment (mattmoor) [inline ISSUE_TEMPLATE/feature-request.md]: Remove trailing whitespace:
```suggestion
**Time Estimate (optional):**
```

**Comment by nachocano** [comment]
@grantr can you also add this to our beloved knative-gcp?

**Comment by grantr** [comment]
https://github.com/google/knative-gcp/pull/559

**Comment by nachocano** [comment]
Thanks!!!

File: ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (matzew) [reply ISSUE_TEMPLATE/feature-request.md]: :-)

**Comment by matzew** [comment]
@grantr when the shit hits the fan ...

https://github.com/knative/eventing-operator/pull/20#issuecomment-548310246

LOLOL

**Comment by lionelvillard** [comment]
Welcome to markdownlint hell

**Comment by grantr** [comment]
Ok the templates are now added to the right directory... :sweat_smile:

File: .github/pull-request-template.md [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
+change has no user-visible impact, no release-note is needed.
+-->
+
 ```release-note
-```
\ No newline at end of file
+
+```
+
+**Docs**
```
Comment (n3wscott) [inline .github/pull-request-template.md]: why is this not commented out ?

Comment (grantr) [reply .github/pull-request-template.md]: This is intended to be a call to action (coupled with the comment below), but maybe not a clear enough one. Suggestions?

**Comment by n3wscott** [comment]
THANKS!
/lgtm

**Comment by n3wscott** [comment]
I thought we turned off markdown linting...

```
------------------------------------
---- Linting the markdown files ----
------------------------------------
---- Thu Feb 20 09:46:05 PST 2020
------------------------------------
.github/ISSUE_TEMPLATE/bug.md:19: MD036 Emphasis used instead of a header

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
.github/issue-template.md:1: MD002 First header should be a top level header

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
.github/pull-request-template.md:5: MD002 First header should be a top level header
.github/pull-request-template.md:7: MD003 Header style
.github/pull-request-template.md:34: MD014 Dollar signs used before commands without showing output
.github/pull-request-template.md:7: MD022 Headers should be surrounded by blank lines
.github/pull-request-template.md:23: MD036 Emphasis used instead of a header
.github/pull-request-template.md:38: MD036 Emphasis used instead of a header
```

**Comment by n3wscott** [comment]
in this pr, add this file to eventing-contrib: https://github.com/knative/eventing/blob/master/test/markdown-lint-config.rc

File: test/markdown-lint-config.rc [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# For help, see
+# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```
Comment (mattmoor) [inline test/markdown-lint-config.rc]: Remove trailing whitespace:
```suggestion
# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```

Comment (grantr) [reply test/markdown-lint-config.rc]: lol linting the markdown lint config

**Comment by grantr** [comment]
This is essentially blocked until https://github.com/knative/test-infra/pull/1726 gets in.

**Comment by grantr** [comment]
Now with #957 the github issue and PR templates won't be markdown linted anymore. This is RFAL.

**Pull request merged**